src/app-bar.jsx

```diff
@@ -127,6 +159,7 @@ const AppBar = React.createClass({
       showMenuIconButton: true,
       title: '',
       zDepth: 1,
+      position: 'fixed',
```

Comment:
What about using `static` as the default?
Should we really introduce this breaking change?